### DISCUSSION TITLE
Restore README file

### DISCUSSION THREAD

**Discussion by Gracetoh** [community/community#27710] [Other Feature Feedback, Questions, & Ideas]
Hi all, 
i have typed finished the entire README file, and have committed it to the main branch. I was then doing my slides and was about to submit my work, when i returned to Github, it states that the repository doesn't contain the 'README.md' path in 'main'???? All my hard work is gone.... please help

**Comment by airtower-luna** [comment]
This sounds like you wrote the README in the browser? In that case I can only think of two things:

* Check if the commit went to another branch, maybe to a PR?
* If you still have the browser tab, go back to the edit. If you're lucky, your browser might still have the text.

Otherwise I'm afraid there isn't much to do. If you did something else, please describe it in more detail.

**Reply by Gracetoh** [reply]
hi, thankyou for replying! yeap i wrote the README in the browser! Am not able to retrieve anything back and this is what my browser is showing:
![Screenshot 2022-07-30 at 11 03 07 PM](https://user-images.githubusercontent.com/104892874/181920234-9b60d8cf-677b-490c-88df-33f8b8403f64.png)

**Reply by airtower-luna** [reply]
Can you open the repository page itself?

**Reply by Gracetoh** [reply]
nope, not at all :(

**Reply by airtower-luna** [reply]
That sounds like either the link is incorrect, the repository has been deleted, or you don't have access to it.

Is the repository your own? You don't have any public repositories, so I can't see. If it's private you need to be logged in.

**Reply by Gracetoh** [reply]
yup the repository its my personal project! hm, i dont think the link is incorrect because i was on that page (did not click away from the page). But when i wanted to submit my files, i clicked onto the same tab and the error message just popped up :(

**Reply by airtower-luna** [reply]
In that case I can think of only a few more things to try:

* Double-check if are logged in.
* Go to your profile, click to the repository tab. See if you can find any other private repositories of yours, and specifically the missing one.
* Check the security log on your account (https://github.com/settings/security-log). I know from experience that creating a repository shows up there, so I'd expect deletion to do the same.

**Reply by Gracetoh** [reply]
i've just checked everything, but still couldn't retrieve it back :/ so weird tho

**Reply by airtower-luna** [reply]
That really is weird. If you can access other private repositories and just that one is missing the only remaining thing I can think of is to [file a bug report with GitHub support](https://support.github.com/contact/bug-report). Repositories shouldn't just go missing, maybe their engineers can figure out what's going on.

**Reply by Gracetoh** [reply]
noted! thank you so much for your help!

**Reply by airtower-luna** [reply]
Good luck! :slightly_smiling_face:

**Comment by jfullstackdev** [comment]
if that is an accidental deleted repo that you may not be aware of, you can restore it

https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository
